internal/services/appservice/windows_function_app_resource.go

```diff
@@ -1263,7 +1288,13 @@ func (m *WindowsFunctionAppModel) unpackWindowsFunctionAppSettings(input web.Str
 			m.BuiltinLogging = true
 
 		case "WEBSITE_VNET_ROUTE_ALL":
-			// Filter out - handled by site_config setting `vnet_route_all_enabled`
+		// Filter out - handled by site_config setting `vnet_route_all_enabled`
+
+		case "WEBSITE_RUN_FROM_PACKAGE":
+			// Keep if user explicitly set, otherwise filter out as will have been added by ADO et al
+			if _, ok := metadata.ResourceData.GetOk("app_settings.WEBSITE_RUN_FROM_PACKAGE"); ok {
+				appSettings[k] = utils.NormalizeNilableString(v)
```

Comment:
The utils package is going to be gradually deprecated, so `pointer.From()` should be used here.
```suggestion
				appSettings[k] = pointer.From(v)
```